.babelrc.js

```diff
@@ -0,0 +1,13 @@
+const { BABEL_ENV, NODE_ENV } = process.env;
+const modules = BABEL_ENV === 'cjs' || NODE_ENV === 'test' ? 'commonjs' : false;
```

Comment:
is this what the cross-env is for?

Reply:
`cross-env` just sets any env variables consistently on all platforms (so it should work also on windows)